### ISSUE TITLE
Deprecation Warning

### ISSUE THREAD

**Issue by amokster88** [Sayrix/Ticket-Bot#84]
I have installed the bot and it is working very nicely, I love it. Can't thank you enough to you DEV keep it up!

So, I am running this bot in a docker container.
In the logs I can see some warnings related to discord button I think, I am attaching the warnings below.
```
(node:1) DeprecationWarning: The SelectMenuBuilder class is deprecated, use StringSelectMenuBuilder instead.
(node:1) DeprecationWarning: The SelectMenuOptionBuilder class is deprecated, use StringSelectMenuOptionBuilder instead.
```

**Comment by Sayrix** [comment]
Thats just some deprecation warning from discord.js you can ignore them

**Comment by amokster88** [comment]
Also the transcript fails to create when there is pics in the chat, the bot hanged, and got stuck in creating transcript and then I had to restart the bot and manually deleted the ticket channel.
The logs showed this at that time.
```

     headers: [AxiosHeaders],
      method: 'post',
      url: 'https://transcript.cf/upload',
      data: '{"buffer":{"type":"Buffer","data":[60,104,116,109,108,62,60,104,101,97,100,62,60,109,101,116,97,32,110,97,109,101,61,34,118,105,101,119,112,111,114,116,34,32,99,111,110,116,101,110,116,61,34,119,105,100,116,104,61,100,101,118,105,99,101,45,119,105,100,116,104,34,47,62,60,108,105,110,107,32,114,101,108,61,34,105,99,111,110,34,32,116,121,112,101,61,34,105,109,97,103,101,47,112,110,103,34,32,104,114,101,102,61,34,104,116,116,112,115,58,47,47,99,100,110,46,100,105,115,99,111,114,100,97,112,112,46,99,111,109,47,105,99,111,110,115,47,57,54,51,55,50,57,50,57,55,48,49,53,55,48,57,55,48,54,47,97,95,53,55,54,50,51,97,55,53,102,57,97,54,52,54,50,53,52,55,55,50,97,102,55,50,98,53,101,52,52,55,56,97,46,103,105,102,63,115,105,122,101,61,49,54,34,47,62,60,116,105,116,108,101,62,116,105,99,107,101,116,45,108,105,103,109,97,45,49,49,60,47,116,105,116,108,101,62,60,115,99,114,105,112,116,62,119,105,110,100,111,119,46,36,100,105,115,99,111,114,100,77,101,115,115,97,103,101,61,123,112,114,111,102,105,108,101,115,58,123,34,57,56,49,56,48,53,53,52,48,51,51,50,49,53,56,57,57,54,34,58,123,34,97,117,116,104,111,114,34,58,34,65,43,32,84,105,99,107,101,116,115,34,44,34,97,118,97,116,97,114,34,58,34,104,116,116,112,115,58,47,47,99,100,110,46,100,105,115,99,111,114,100,97,112,112,46,99,111,109,47,97,118,97,116,97,114,115,47,57,56,49,56,48,53,53,52,48,51,51,50,49,53,56,57,57,54,47,54,49,100,98,53,56,54,100,56,57,52,98,49,48,55,102,101,98,50,54,50,57,97,56,49,52,99,50,97,97,48,54,46,119,101,98,112,63,115,105,122,101,61,54,52,34,44,34,114,111,108,101,67,111,108,111,114,34,58,34,35,102,102,102,102,102,102,34,44,34,114,111,108,101,73,99,111,110,34,58,34,104,116,116,112,115,58,47,47,99,100,110,46,100,105,115,99,111,114,100,97,112,112,46,99,111,109,47,114,111,108,101,45,105,99,111,110,115,47,57,57,49,57,54,48,50,50,56,53,53,51,56,50,54,51,49,52,47,98,53,102,50,97,52,57,52,98,54,53,53,98,100,57,55,56,102,52,53,57,55,102,48,98,100,50,98,101,52,52,52,46,119,101,98,112,34,44,34,98,111,116,34,58,116,114,117,101,44,34,118,101,114,105,102,105,101,100,34,58,102,97,108,115,101,125,44,34,54,57,56,56,52,49,56,49,48,53,57,53,56,55,50,55,57,57,34,58,123,34,97,117,116,104,111,114,34,58,34,91,83,84,65,70,70,93,32,85,110,99,108,101,32,76,117,107,101,226,132,162,34,44,34,97,118,97,116,97,114,34,58,34,104,116,116,112,115,58,47,47,99,100,110,46,100,105,115,99,111,114,100,97,112,112,46,99,111,109,47,103,117,105,108,100,115,47,57,54,51,55,50,57,50,57,55,48,49,53,55,48,57,55,48,54,47,117,115,101,114,115,47,54,57,56,56,52,49,56,49,48,53,57,53,56,55,50,55,57,57,47,97,118,97,116,97,114,115,47,97,95,101,52,54,100,56,55,99,98,49,51,56,57,52,49,52,98,49,50,100,50,51,50,102,57,102,51,102,55,100,102,53,57,46,103,105,102,63,115,105,122,101,61,54,52,34,44,34,114,111,108,101,67,111,108,111,114,34,58,34,35,102,48,101,100,48,48,34,44,34,114,111,108,101,73,99,111,110,34,58,34,104,116,116,112,115,58,47,47,99,100,110,46,100,105,115,99,111,114,100,97,112,112,46,99,111,109,47,114,111,108,101,45,105,99,111,110,115,47,57,54,52,49,48,49,55,55,49,53,52,48,54,57,55,48,57,56,47,100,52,50,98,98,102,57,57,57,55,51,48,97,102,52,101,54,100,55,48,48,53,101,52,50,49,57,98,50,99,97,50,46,119,101,98,112,34,44,34,114,111,108,101,78,97,109,101,34,58,34,203,154,226,130,138,226,128,167,239,184,182,234,146,183,234,146,166,32,226,157,157,32,65,68,77,73,78,225,181,142,32,226,157,156,32,224,173,167,32,226,138,185,226,130,138,234,146,183,34,44,34,98,111,116,34,58,102,97,108,115,101,44,34,118,101,114,105,102,105,101,100,34,58,102,97,108,115,101,125,44,34,49,53,53,49,52,57,49,48,56,49,56,51,54,57,53,51,54,48,34,58,123,34,97,117,116,104,111,114,34,58,34,240,159,164,150,32,65,43,32,65,73,32,66,79,84,34,44,34,97,118,97,116,97,114,34,58,34,104,116,116,112,115,58,47,47,99,100,110,46,100,105,115,99,111,114,100,97,112,112,46,99,111,109,47,97,118,97,116,97,114,115,47,49,53,53,49,52,57,49,48,56,49,56,51,54,57,53,51,54,48,47,49,57,97,53,101,101,52,49,49,52,98,52,55,49,57,53,102,99,101,99,99,54,54,52,54,102,50,51,56,48,98,49,46,119,101,98,112,63,115,105,122,101,61,54,52,34,44,34,114,111,108,101,67,111,108,111,114,34,58,34,35,102,102,102,102,102,102,34,44,34,114,111,108,101,73,99,111,110,34,58,34,104,116,116,112,115,58,47,47,99,100,110,46,100,105,115,99,111,114,100,97,112,112,46,99,111,109,47,114,111,108,101,45,105,99,111,110,115,47,57,55,51,52,48,52,48,49,52,55,51,49,52,57,51,52,48,54,47,56,57,99,50,50,100,98,52,54,98,99,51,48,100,97,53,50,53,100,101,53,48,54,51,50,97,98,49,57,55,50,56,46,119,101,98,112,34,44,34,98,111,116,34,58,116,114,117,101,44,34,118,101,114,105,102,105,101,100,34,58,116,114,117,101,125,44,34,50,56,49,57,51,50,50,56,54,55,50,57,57,49,50,51,50,48,34,58,123,34,97,117,116,104,111,114,34,58,34,76,105,103,109,97,34,44,34,97,118,97,116,97,114,34,58,34,104,116,116,112,115,58,47,47,99,100,110,46,100,105,115,99,111,114,100,97,112,112,46,99,111,109,47,97,118,97,116,97,114,115,47,50,56,49,57,51,50,50,56,54,55,50,57,57,49,50,51,50,48,47,53,101,50,57,102,50,51,49,48,52,57,51,57,49,102,56,53,97,54,50,98,48,97,55,56,97,98,57,102,54,98,99,46,119,101,98,112,63,115,105,122,101,61,54,52,34,44,34,114,111,108,101,67,111,108,111,114,34,58,34,35,100,102,57,55,49,49,34,44,34,114,111,108,101,73,99,111,110,34,58,34,104,116,116,112,115,58,47,47,99,100,110,46,100,105,115,99,111,114,100,97,112,112,46,99,111,109,47,114,111,108,101,45,105,99,111,110,115,47,49,48,48,57,56,55,49,49,54,51,53,55,48,50,55,48,50,51,56,47,97,98,55,101,49,101,55,57,50,57,49,49,49,52,98,49,102,52,100,56,102,100,98,53,49,54,100,57,54,53,100,54,46,119,101,98,112,34,44,34,114,111,108,101,78,97,109,101,34,58,34,65,43,226,132,162,32,80,45,83,104,97,114,101,34,44,34,98,111,116,34,58,102,97,108,115,101,44,34,118,101,114,105,102,105,101,100,34,58,102,97,108,115,101,125,125,125,60,47,115,99,114,105,112,116,62,60,115,99,114,105,112,116,62,100,111,99,117,109,101,110,116,46,97,100,100,69,118,101,110,116,76,105,115,116,101,110,101,114,40,34,99,108,105,99,107,34,44,116,61,62,123,108,101,116,32,101,61,116,46,116,97,114,103,101,116,59,105,102,40,33,101,41,114,101,116,117,114,110,59,108,101,116,32,111,61,101,63,46,103,101,116,65,116,116,114,105,98,117,116,101,40,34,100,97,116,97,45,103,111,116,111,34,41,59,105,102,40,111,41,123,108,101,116,32,114,61,100,111,99,117,109,101,110,116,46,103,101,116,69,108,101,109,101,110,116,66,121,73,100,40,96,109,45,36,123,111,125,96,41,59,114,63,40,114,46,115,99,114,111,108,108,73,110,116,111,86,105,101,119,40,123,98,101,104,97,118,105,111,114,58,34,115,109,111,111,116,104,34,44,98,108,111,99,107,58,34,99,101,110,116,101,114,34,125,41,44,114,46,115,116,121,108,101,46,98,97,99,107,103,114,111,117,110,100,67,111,108,111,114,61,34,114,103,98,97,40,49,52,56,44,32,49,53,54,44,32,50,52,55,44,32,48,46,49,41,34,44,114,46,115,116,121,108,101,46,116,114,97,110,115,105,116,105,111,110,61,34,98,97,99,107,103,114,111,117,110,100,45,99,111,108,111,114,32,48,46,53,115,32,101,97,115,101,34,44,115,101,116,84,105,109,101,111,117,116,40,40,41,61,62,123,114,46,115,116,121,108,101,46,98,97,99,107,103,114,111,117,110,100,67,111,108,111,114,61,34,116,114,97,110,115,112,97,114,101,110,116,34,125,44,49,101,51,41,41,58,99,111,110,115,111,108,101,46,119,97,114,110,40,34,77,101,115,115,97,103,101,32,36,123,103,111,116,111,125,32,110,111,116,32,102,111,117,110,100,46,34,41,125,125,41,59,60,47,115,99,114,105,112,116,62,60,115,99,114,105,112,116,32,116,121,112,101,61,34,109,111,100,117,108,101,34,32,115,114,99,61,34,104,116,116,112,115,58,47,47,99,100,110,46,106,115,100,101,108,105,118,114,46,110,101,116,47,110,112,109,47,64,100,101,114,111,99,107,100,101,118,47,100,105,115,99,111,114,100,45,99,111,109,112,111,110,101,110,116,115,45,99,111,114,101,64,94,51,46,53,46,49,47,100,105,115,116,47,100,101,114,111,99,107,100,101,118,45,100,105,115,99,111,114,100,45,99,111,109,112,111,110,101,110,116,115,45,99,111,114,101,47,100,101,114,111,99,107,100,101,118,45,100,105,115,99,111,114,100,45,99,111,109,112,111,110,101,110,116,115,45,99,111,114,101,46,101,115,109,46,106,115,34,62,60,47,115,99,114,105,112,116,62,60,47,104,101,97,100,62,60,98,111,100,121,32,115,116,121,108,101,61,34,109,97,114,103,105,110,58,48,59,109,105,110,45,104,101,105,103,104,116,58,49,48,48,118,104,34,62,60,100,105,115,99,111,114,100,45,109,101,115,115,97,103,101,115,32,115,116,121,108,101,61,34,109,105,110,45,104,101,105,103,104,116,58,49,48,48,118,104,34,62,60,100,105,115,99,111,114,100,45,104,101,97,100,101,114,32,103,117,105,108,100,61,34,65,43,32,77,69,68,73,65,226,132,162,34,32,99,104,97,110,110,101,108,61,34,116,105,99,107,101,116,45,108,105,103,109,97,45,49,49,34,32,105,99,111,110,61,34,104,116,116,112,115,58,47,47,99,100,110,46,100,105,115,99,111,114,100,97,112,112,46,99,111,109,47,105,99,111,110,115,47,57,54,51,55,50,57,50,57,55,48,49,53,55,48,57,55,48,54,47,97,95,53,55,54,50,51,97,55,53,102,57,97,54,52,54,50,53,52,55,55,50,97,102,55,50,98,53,101,52,52,55,56,97,46,103,105,102,63,115,105,122,101,61,49,50,56,34,62,84,104,105,115,32,105,115,32,116,104,101,32,115,116,97,114,116,32,111,102,32,35,116,105,99,107,101,116,45,108,105,103,109,97,45,49,49,32,99,104,97,110,110,101,108,46,60,47,100,105,115,99,111,114,100,45,104,101,97,100,101,114,62,60,100,105,115,99,111,114,100,45,109,101,115,115,97,103,101,32,105,100,61,34,109,45,49,48,53,51,55,49,53,56,56,49,49,53,50,50,49,55,49,52,56,34,32,101,100,105,116,101,100,61,34,102,97,108,115,101,34,32,104,105,103,104,108,105,103,104,116,61,34,102,97,108,115,101,34,32,112,114,111,102,105,108,101,61,34,57,56,49,56,48,53,53,52,48,51,51,50,49,53,56,57,57,54,34,62,60,100,105,115,99,111,114,100,45,109,101,110,116,105,111,110,32,116,121,112,101,61,34,117,115,101,114,34,62,76,105,103,109,97,60,47,100,105,115,99,111,114,100,45,109,101,110,116,105,111,110,62,60,100,105,115,99,111,114,100,45,101,109,98,101,100,32,101,109,98,101,100,45,116,105,116,108,101,61,34,84,105,99,107,101,116,32,71,101,110,101,114,97,108,32,83,117,112,112,111,114,116,34,32,115,108,111,116,61,34,101,109,98,101,100,115,34,32,99,111,108,111,114,61,34,35,48,48,102,102,48,48,34,62,60,100,105,115,99,111,114,10'... 26713101 more characters
    },
    request: ClientRequest {
      _events: [Object: null prototype],
      _eventsCount: 7,
      _maxListeners: undefined,
      outputData: [],
      outputSize: 0,
      writable: true,
      destroyed: false,
      _last: true,
      chunkedEncoding: false,
      shouldKeepAlive: false,
      maxRequestsOnConnectionReached: false,
      _defaultKeepAlive: true,
      useChunkedEncodingByDefault: true,
      sendDate: false,
      _removedConnection: false,
      _removedContLen: false,
      _removedTE: false,
      strictContentLength: false,
      _contentLength: '26723101',
      _hasBody: true,
      _trailer: '',
      finished: true,
      _headerSent: true,
      _closed: false,
      socket: [TLSSocket],
      _header: 'POST /upload HTTP/1.1\r\n' +
        'Accept: application/json, text/plain, */*\r\n' +
        'Content-Type: application/json\r\n' +
        'User-Agent: axios/1.2.1\r\n' +
        'Content-Length: 26723101\r\n' +
        'Accept-Encoding: gzip, compress, deflate, br\r\n' +
        'Host: transcript.cf\r\n' +
        'Connection: close\r\n' +
        '\r\n',
      _keepAliveTimeout: 0,
      _onPendingData: [Function: nop],
      agent: [Agent],
      socketPath: undefined,
      method: 'POST',
      maxHeaderSize: undefined,
      insecureHTTPParser: undefined,
      path: '/upload',
      _ended: true,
      res: [IncomingMessage],
      aborted: false,
      timeoutCb: null,
      upgradeOrConnect: false,
      parser: null,
      maxHeadersCount: null,
      reusedSocket: false,
      host: 'transcript.cf',
      protocol: 'https:',
      _redirectable: [Writable],
      [Symbol(kCapture)]: false,
      [Symbol(kBytesWritten)]: 0,
      [Symbol(kEndCalled)]: true,
      [Symbol(kNeedDrain)]: false,
      [Symbol(corked)]: 0,
      [Symbol(kOutHeaders)]: [Object: null prototype],
      [Symbol(kUniqueHeaders)]: null
    },
    response: {
      status: 413,
      statusText: 'Payload Too Large',
      headers: [AxiosHeaders],
      config: [Object],
      request: [ClientRequest],
      data: [Object]
    }
  }
} undefined
```

**Comment by Pogysek** [comment]
Hi, did you work it out or?

**Comment by amokster88** [comment]
still in large chats with multiple images it fails

**Comment by Sayrix** [comment]
Hi, about the deprecation warnings thats fixed in the version 2.0.1, for your error when you create a transcript, please give more details like the complete copy&paste of your console ect

**Comment by thomasbnt** [comment]
Resolved from this commit https://github.com/Sayrix/Ticket-Bot/commit/1fcabbbd4fda22bab738bee4f2619bafe3a8402e
